packages/wrangler/src/deployment-bundle/bundle.ts

```diff
@@ -507,6 +512,18 @@ export async function bundleWorker(
 		);
 	}
 
+	const notExportedWorkflows = workflowBindings
+		.filter((x) => !x.script_name && !entryPoint.exports.includes(x.class_name))
+		.map((x) => x.class_name);
+	if (notExportedWorkflows.length) {
+		const relativePath = path.relative(process.cwd(), entryFile);
```

Comment:
Is this always going to be relative to the cwd? We have access to the `Entry` here too, so we could feasibly use that instead.

Reply:
Good call

Reply:
Seems entryFile _is_ entry.file